tests/PHPStan/Rules/Comparison/ImpossibleCheckTypeFunctionCallRuleTest.php

```diff
@@ -181,30 +181,30 @@ public function testImpossibleCheckTypeFunctionCall(): void
 					631,
 				],
 				[
-					'Call to function method_exists() with \'CheckTypeFunctionCall\\\\MethodExistsWithTrait\' and \'method\' will always evaluate to true.',
+					'Call to function method_exists() with class-string<static(CheckTypeFunctionCall\MethodExistsWithTrait)> and \'method\' will always evaluate to true.',
```

Comment:
I think the previous message was better, but since there is already `$this(...)` in the messages above, I decided to ignore it.